rtil/src/threads/ue/rebo/mod.rs

```diff
@@ -46,6 +47,7 @@ struct State {
     player_minimap_image: RgbaImage,
     // will keep textures forever, even if the player doesn't exist anymore, but each texture is only a few MB
     player_minimap_textures: HashMap<Rgba<u8>, UTexture2D>,
+    livesplit_state: LiveSplit,
```

Comment:
Actually, now that `livesplit` is it's own submodule of `rebo`, it might make sense to have it in its separate static again (sorry for the back-and-forth).